### DISCUSSION TITLE
Sublime Text Key Mapping with Neovintageous

### DISCUSSION THREAD

**Discussion by MichaelMFox** [gerardroche/gerardroche.github.io#54] [General]
Is this the right way to obtain ctrl-[ for escape (to command mode) for Neovintageous ?

{"keys": ["ctrl+["], "command": "_enter_normal_mode", "context":
            [
                { "key": "setting.command_mode", "operand": false },
                { "key": "setting.is_widget", "operand": false }
            ]
    },

**Comment by gerardroche** [comment]
Duplicate of https://github.com/gerardroche/gerardroche.github.io/issues/3
